src/publisher.js

```diff
@@ -0,0 +1,29 @@
+var mqtt = require('mqtt')
+var client  = mqtt.connect('tcp://aerostun.dev:1883')
+  
+function forward(){
+  client.publish('/smartcar/control/throttle', '50')
+}
+
+function backward(){
+  client.publish('/smartcar/control/throttle', '-100')
+}
+
+function left(){
+  client.publish('/smartcar/control/steering', '-20')
+}
+
+function right(){
+  client.publish('/smartcar/control/steering', '20')
+}
```

Comment:
These perhaps should take the value as an input or read it from a variable instead? Looks good otherwise though!